base/mac/BUILD.gn

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+if (is_mac) {
+  import("//build/config/mac/rules.gni")
```

Comment:
I don't think you need any of these imports

Reply:
Removed.